packages/storybook8/stories/Concepts/RealTimeText/Docs.mdx

```diff
@@ -89,3 +67,27 @@ Note that Real-Time Text will be enabled for all participants in the call once t
 
 To read more details about the underlying infrastructure, click here:  
 [Real Time Text (RTT) Overview - An Azure Communication Services concept document | Microsoft Learn](https://learn.microsoft.com/en-us/azure/communication-services/concepts/voice-video-calling/real-time-text)
+
+## FAQ
+
+### How can I enable RTT?
+
+RTT can be enabled during calls via the control bar “more” button.
+
+Enabling RTT opens the RTT window for everyone in the call. When one user enables RTT, other users do not need to enable it for themselves. This window cannot be closed for the duration of the call nor can RTT be disabled.
+
+### What versions of UI Library support RTT?
+
+RTT is currently in public preview and supported in 1.24.0-beta.1.
+
+### When I type a sentence in the RTT window, when does it get sent?
+
+A message is finalized (or "committed") when the user explicitly takes an action, such as pressing "Enter," or if no additional text is typed for 3 seconds, at which point the text will automatically commit.
```

Comment:
the question asks about "sent"
But the answer never mentions sent. Rather it mentions "Finalized" and "committed". Should we somewhere indicate sent = finalized? If that's what they are specifically asking here.

Reply:
Let me check with Tay on this

Reply:
Updated